src/main/java/classify/ui/UI.java

```diff
@@ -61,8 +61,8 @@ public class UI {
     private static final String DELETE_UNDONE_MESSAGE = "Last delete undone!";
     private static final String STUDENT_RESTORED_MESSAGE = "Student has been restored!";
     private static final String NO_RECENT_DELETES = "No recent deletes found!";
-    private static final String STUDENTS_IN_ARCHIVE = "3. List of students in archive";
-    private static final String STUDENTS_IN_DELETED = "4. List of students in recently deleted";
+    private static final String STUDENTS_IN_ARCHIVE = "4. List of students in archive";
```

Comment:
rewritten to group similar options together, excellent decision to make the experience more intuitive for the user!